From: Sam Hocevar <sam+deb@zoy.org>
Date: Wed, 30 Aug 2023 15:19:30 +0200
Subject: Init the package with libquicktime1 instead of libquicktime so

Last-Update: 2011-04-29
Forwarded: not-needed

that we don't conflict with libquicktime0 packages and pass -version-number
to libtool.
Forwarded: not-needed
---
 configure.ac    | 2 +-
 src/Makefile.am | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index da1dabc..1836baa 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 AC_INIT(src/lqt_quicktime.c)
-AM_INIT_AUTOMAKE(libquicktime, 1.2.4)
+AM_INIT_AUTOMAKE(libquicktime2, 1.2.4)
 
 LQT_VERSION=$VERSION
 
diff --git a/src/Makefile.am b/src/Makefile.am
index fadd879..41bd94f 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -9,7 +9,7 @@ AM_CFLAGS = -DLOCALE_DIR=\"$(localedir)\"
 #
 libquicktime_la_LIBADD = @ICONV_LIBS@ @LTLIBINTL@
 
-libquicktime_la_LDFLAGS = -export-dynamic
+libquicktime_la_LDFLAGS = -export-dynamic -version-number 2
 
 # build only the files, which can be compiled
 
